test/cbmc/proofs/DHCPv6/DHCPv6ProcessEndPoint/DHCPv6ProcessEndPoint_harness.c

```diff
@@ -0,0 +1,74 @@
+/* Standard includes. */
+#include <stdint.h>
+
+/* FreeRTOS includes. */
+#include "FreeRTOS.h"
+#include "task.h"
+#include "semphr.h"
+
+/* FreeRTOS+TCP includes. */
+#include "FreeRTOS_IP.h"
+#include "FreeRTOS_Sockets.h"
+#include "FreeRTOS_IP_Private.h"
+#include "FreeRTOS_UDP_IP.h"
+#include "FreeRTOS_DHCP.h"
+
+/* CBMC includes. */
+#include "cbmc.h"
+
+
+
+/* Static members defined in FreeRTOS_DHCP.c */
+extern Socket_t xDHCPv6Socket;
+void prvCreateDHCPv6Socket( NetworkEndPoint_t * pxEndPoint );
+void prvSendDHCPMessage( NetworkEndPoint_t * pxEndPoint );
+
+BaseType_t __CPROVER_file_local_FreeRTOS_DHCPv6_c_xDHCPv6ProcessEndPoint_HandleState( NetworkEndPoint_t * pxEndPoint,
+                                                                                      DHCPMessage_IPv6_t * pxDHCPMessage )
+{
+    __CPROVER_assume( pxEndPoint != NULL );
+    __CPROVER_assume( pxDHCPMessage != NULL );
```

Comment:
I think you're checking the variable in this function. If so, you might need `__CPROVER_assert` instread of `__CPROVER_assume`.